api/Create.inc

```diff
@@ -266,7 +280,7 @@ class Create implements ActionInterface {
    * @return DOMNode
    *   The DOMNode that is created.
    */
-  protected function createXML(XMLDocument $document, $value) {
+  protected function createXml(XMLDocument $document, DOMElement $value) {
```

Comment:
Looks like this can be mixed: 

{code}
Recoverable fatal error: Argument 2 passed to Create::createXml() must be an instance of DOMElement, string given, called in /var/www/drupal8/modules/contrib/xml_forms/api/Create.inc on line 209 and defined in Create->createXml() (line 283 of /var/www/drupal8/modules/contrib/xml_forms/api/Create.inc).
{code}

https://github.com/discoverygarden/xml_forms/pull/3/files#diff-e2603a74c5db45f9292b39facf32d512R169